index.html

```diff
@@ -0,0 +1,515 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<!--
+ _________   _______       ___     ___    ___      _____      _                  _      _ _
+|         | |        \    |   |   |   |  |   |    / ____|    | |                | |    | (_)
+|__     __| |    _    \   |   |   '   |  '   '   | |    _   _| |__   ___ _ __ __| | ___| |_  __ _
+   |   |    |   |  \   \  |   '_ /    '_/   /    | |   | | | | '_ \ / _ \ '__/ _` |/ _ \ | |/ _` |
+   |   |    |   |   |   | |         /      /     | |___| |_| | |_) |  __/ | | (_| |  __/ | | (_| |
+   '___'    '___'   '___' '_______ / '____/       \_____\__, |_.__/ \___|_|  \__,_|\___|_|_|\__,_|
+                                                         __/ |
+                                                        |___/
+
+-->
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>TNW</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+  <header>
```

Comment:
This looks nothing like the original when the width is ~764px

Reply:
@heyalvaro sorry I don't quite understand your comments. Not sure what you meant by pointing to the code snippets. Did you mean the "header" element in this case? So, you are pointing at elements?
Also, does this "~764px" mean "approximately" 764px?